zkevm-circuits/src/witness/tx.rs

```diff
@@ -42,66 +43,71 @@ impl Transaction {
     /// calldata) and tx_calldata
     pub fn table_assignments<F: Field>(
         &self,
-        challenges: Challenges<Value<F>>,
-    ) -> [Vec<[Value<F>; 4]>; 2] {
+        _challenges: Challenges<Value<F>>,
```

Comment:
This challenge can go too.
```suggestion
```

Reply:
Have clean up the _randomness var in commit https://github.com/privacy-scaling-explorations/zkevm-circuits/commit/0646b790010306d9df41cd27002bf9890d017202